package-lock.json

```diff
@@ -20084,7 +20084,7 @@
           "dependencies": {
             "get-stream": {
               "version": "3.0.0",
-              "resolved": "https://registry.npmjs.org/get-stream/-/get-stream-3.0.0.tgz",
+              "resolved": "http://registry.npmjs.org/get-stream/-/get-stream-3.0.0.tgz",
```

Comment:
Could you please revert `package-lock.json` and re-test after `npm ci`?

Reply:
@olshansky @Betrozov Why should I revert this file if it doesn't add to `.gitignore`? I have just run `npm i`.

Reply:
package-lock.json is not ignored intentionally, we need it to keep all of us with the same dependencies.
we always run `npm ci` in such cases.